vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/IssuerAgent.kt

```diff
@@ -113,14 +138,15 @@ class IssuerAgent(
                 signed = issuanceDate,
                 validFrom = issuanceDate,
                 validUntil = expirationDate,
-            )
+            ),
+            status = credentialStatus
         )
         val issuerSigned = IssuerSigned.fromIssuerSignedItems(
             namespacedItems = mapOf(credential.scheme.isoNamespace!! to credential.issuerSignedItems),
             issuerAuth = coseService.createSignedCose(
                 payload = mso,
                 serializer = MobileSecurityObject.serializer(),
-                addKeyId = false,
+                addKeyId = true,
```

Comment:
Is this change necessary?

Reply:
No, I just needed to change a test to use a key with self signed cert 
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b